src/components/Categories.js

```diff
@@ -0,0 +1,9 @@
+function Categories() {
+  return (
+    <>
+
```

Comment:
- [ ] [optional] @Tracy-miranja great work implementing the changes requested, however, the last reviewer forgot to point out that in the categories page you need a button that checks for status (it doesn't work now but acts as a placeholder for future features implementation)

Reply:
Hi @danmainah thanks for the feedback, ill check on that.